nvflare/fuel_opt/utils/omegaconf_loader.py

```diff
@@ -13,19 +13,21 @@
 # limitations under the License.
 from typing import Dict, Optional
 
-from omegaconf import DictConfig, OmegaConf
-
 from nvflare.fuel.utils.config import Config, ConfigFormat, ConfigLoader
 
 
 class OmegaConfConfig(Config):
-    def __init__(self, conf: DictConfig, file_path: Optional[str] = None):
+    def __init__(self, conf, file_path: Optional[str] = None):
         super(OmegaConfConfig, self).__init__(conf, ConfigFormat.OMEGACONF, file_path)
 
     def to_dict(self, resolve: Optional[bool] = True) -> Dict:
+        from omegaconf import OmegaConf
```

Comment:
for these lines, do we want to instead do optional import at the top or something?